package.json

```diff
@@ -203,7 +204,9 @@
     "clean-up-svg-icons": "node ./scripts/clean-up-svg-icons.js",
     "collect-svg-icons": "node scripts/collect-svg-icons.js",
     "check-dependencies": "ckeditor5-dev-dependency-checker",
-    "check-dependencies:versions-match": "node ./scripts/ci/check-dependencies-versions-match.js"
+    "check-dependencies:versions-match": "node ./scripts/ci/check-dependencies-versions-match.js",
+    "check-theme-lark-imports": "node ./scripts/check-theme-lark-imports.js",
+    "build:dist": "node ./scripts/build-ckeditor5.mjs"
   },
   "lint-staged": {
     "**/*.{js,ts}": [
```

Comment:
Since we have `*.mjs` files now, it would be good to add them to the `lint-staged`.

Reply:
Done :heavy_check_mark: